ethcore/sync/src/sync_io.rs

```diff
@@ -40,9 +41,9 @@ pub trait SyncIo {
 	fn chain(&self) -> &BlockChainClient;
 	/// Get the snapshot service.
 	fn snapshot_service(&self) -> &SnapshotService;
-	/// Returns peer identifier string
-	fn peer_info(&self, peer_id: PeerId) -> String {
-		peer_id.to_string()
+	/// Returns peer version identifier
+	fn peer_info(&self, peer_id: PeerId) -> ClientVersion {
```

Comment:
let's rename it to `peer_version` since it doesn't really return the Id any more